DXRFixes/DeusEx/Classes/Carcass.uc

```diff
@@ -216,15 +219,35 @@ function Frob(Actor Frobber, Inventory frobWith)
 
     if (Inventory != None && player != None)
     {
+        datastorage = class'DataStorage'.static.GetObj(class'DXRando'.default.dxr);
+
         item = Inventory;
         startItem = item;
 
         do
         {
             //player.ClientMessage(self@"=> item="$item);
             nextItem = item.Inventory;
-            if(TryLootItem(player, item))
+
```

Comment:
Shouldn't this all just be within TryLootItem?

Reply:
I agree. It should be there, and the logic becomes easier to follow if is.